### ISSUE TITLE
Color contrast on search box needs update

### ISSUE THREAD

**Issue by ryelle** [ryelle/Foxhound#70]
> Almost everything is fine, but the search widget is not visible to low contrast; the border gives a contrast of 1.34:1 (https://www.joedolson.com/tools/color-contrast.php?type=rgb&color=43%2C36%2C36&color2=252%2C251%2C248&alpha=.15 ), and the placeholder text is also below color contrast. It's acceptable for placeholder text to be under contrast, because it needs to be clear that text is not an entered search term; but *something* of the search form must be visible. As this currently is, somebody with poor color perception may not be able to find the search form at all.

**Comment by ryelle** [comment]
@melchoyce could use some help with this one – the search border needs more color-contrast, but if I just up it to a passing value it looks silly. Maybe the search style should be redesigned?

Current:

![screen shot 2017-02-18 at 9 27 53 am](https://cloud.githubusercontent.com/assets/541093/23093894/90319248-f5bc-11e6-8903-2ae9921143fb.png)

and when the search button is focused it appears like this (see #68):

![screen shot 2017-02-18 at 9 30 03 am](https://cloud.githubusercontent.com/assets/541093/23093903/e2185e48-f5bc-11e6-8933-24b5914f9947.png)

**Comment by melchoyce** [comment]
It definitely feels super dark, but maybe we could reuse `#6a7379`: 

<img width="706" alt="screen shot 2017-02-18 at 12 43 50 pm" src="https://cloud.githubusercontent.com/assets/2846578/23095444/eb1dd610-f5d7-11e6-8cfc-595054f498b9.png">

I'm not sure there's a better solution here. I tried just adding a bottom border, and tried changing the background color, and it didn't look good:

<img width="709" alt="screen shot 2017-02-18 at 12 46 23 pm" src="https://cloud.githubusercontent.com/assets/2846578/23095478/49b45fb4-f5d8-11e6-964b-af9e0e1a3792.png">

<img width="698" alt="screen shot 2017-02-18 at 12 44 59 pm" src="https://cloud.githubusercontent.com/assets/2846578/23095466/23c3f83c-f5d8-11e6-931b-bc5714f7d5bd.png">

Maybe if the `border-bottom` version had a search icon...?

**Comment by ryelle** [comment]
How's this, stealing from the comment form styles:

In a widget, the second shows the submit button focused (should only happen if you tab to it from the search bar)

![screen shot 2017-02-19 at 11 20 27 am](https://cloud.githubusercontent.com/assets/541093/23104109/82b57916-f695-11e6-826f-e2ff945e7fd1.png)

![screen shot 2017-02-19 at 11 20 32 am](https://cloud.githubusercontent.com/assets/541093/23104106/82b45284-f695-11e6-8043-bf6796cf9ace.png)

<hr />

In the search header:

![screen shot 2017-02-19 at 11 20 38 am](https://cloud.githubusercontent.com/assets/541093/23104107/82b4e884-f695-11e6-9076-8d452142372e.png)

![screen shot 2017-02-19 at 11 20 42 am](https://cloud.githubusercontent.com/assets/541093/23104108/82b52254-f695-11e6-91f8-5abe1d576df8.png)

Having the label with adequate contrast should be enough to show where the search for is, and we can keep the decorative border at a lower (failing) contrast.

**Comment by melchoyce** [comment]
I think if you ditch the border-radius on the button I think it's good to go 👍
